packages/hs-react-ui/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import styled from 'styled-components';
+import { storiesOf, addDecorator } from '@storybook/react';
+import { boolean, number } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { withDesign } from 'storybook-addon-designs';
+import { Dropdown } from './Dropdown';
+
+addDecorator(withDesign);
+const design = {
+    type: 'figma',
+    url: 'https://www.figma.com/file/3r2G00brulOwr9j7F6JF59/Generic-UI-Style?node-id=102%3A28',
+};
+
+const Background = styled.div`background: white; height: 100%;`;
```

Comment:
Have you pulled from master? I feel like this isn't necessary anymore.

But either way it stops people from being able to use the storybook background feature to see what the dropdown looks like on a dark background

Reply:
Nope I forgot to remove this.